tools/cloud-build/daily-tests/ansible_playbooks/slurm-integration-test.yml

```diff
@@ -155,12 +155,21 @@
 
     ## Cleanup and fail gracefully
     rescue:
+    - name: Gather logs
+      ansible.builtin.include_tasks:
+        file: tasks/gather_startup_script_logs.yml
+        apply:
+          delegate_to: localhost
+      vars:
+        terraform_apply_stderr: "{{ terraform_output.results.1.stderr }}"
     - name: Include rescue from terraform failure
-      ansible.builtin.include_tasks: "tasks/rescue_terraform_failure.yml"
```

Comment:
You have not added back in the fail out functionality. that was originally contained in this file. IIUC We will now show slurm tests passing that have actually failed. Please fix.

Reply:
Done